app/views/standings/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%
+=begin%>
+ <div class="bg-emerald-500 w-3 h-3 rounded-full"></div> 
+<%
+=end%>
```

Comment:
Why the `begin` and `end` here?

I don't think this is even rendering..?

Reply:
It is what happened when I commented the line out using VSC. 

I want to keep the small ball code for visually showing the result of the last matches.

Today I found [this](https://codepen.io/robstinson/pen/PoGyRgy) which is exactly what I wanted.

Reply:
Solved in #21